app/scripts/models/auth_brokers/fx-desktop-v2.js

```diff
@@ -64,6 +65,16 @@ define(function (require, exports, module) {
         .then(function () {
           return proto.afterCompleteResetPassword.call(self, account);
         });
+    },
+    fetch: function () {
+      var self = this;
+      return proto.fetch.call(self)
+        .then(function () {
+          if (! self.environment.isAboutAccounts()) {
+            // if force_auth is not framed then we do not want to halt
+            self.setBehavior('afterForceAuth', new NullBehavior());
```

Comment:
for functional tests: 

```
return FunctionalHelpers.openPage(self, PAGE_URL, '#fxa-signin-header')
``` 

needs an `options` block with `aboutAccounts: true` will mean that `isAboutAccounts` should return `true`.